CODEWALK.md

```diff
@@ -346,6 +346,118 @@ essentially who is storing what data, for what fee and which sectors have been s
 The keystore contains the binary encoded peer key for interacting securely over the network. 
 This data lives in a file at `$HOME/.filecoin/keystore/self`.
 
+## libp2p Networking
+
+As mentioned above, Filecoin relies on [libp2p](https://libp2p.io/) for its
+networking needs. Here we'll explain some of the mechanics that drive the
+network layer, in particular:
```

Comment:
Please format 1 sentence per line, breaking only very long sentences at a phrase boundary. It makes code review comments easier to target and reduces manual re-wrapping when things change.

Bonus: add a comment at the top describing this format for future contributors.

Reply:
Sure, this format is a bit weird, but I'll reformat. Please make sure to document it so that future contributors don't trip up, and you have an actual style guide to reference.